pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/TransformContext.java

```diff
@@ -76,7 +76,7 @@ public TransformContext(Context context, Object value) {
     }
   }
 
-  public void send() throws IOException {
+  public Record<Object> send() throws IOException {
```

Comment:
Any chance you could use a generic type here? `Record<T>`

Reply:
It must be the same type as TransformFunction's return type so I don't think we can make it a generic